### ISSUE TITLE
大佬，要是有分时图和实时更新数据就完美了

### ISSUE THREAD

**Issue by LvJson** [zhaojijin/flutter_kline#6]

**Comment by kavinda1995** [comment]
I added realtime functionality, but the new bars are coming at the start of the chart. Any idea to resolve that?


我添加了实时功能，但是新的条形图出现在图表的开头。有解决的办法吗？

(I don't know whether the Chinese translate is correct or not :D)

Anyway, nice chart mate!

**Comment by LvJson** [comment]
那个应该只是数据位置问题，把数据翻转过来就可以了吧

**Comment by kavinda1995** [comment]
Yeah that solved. What i did was sort the array to descending order. Btw why it must be like that?

**Comment by LvJson** [comment]
我这里有个更好的K线框架：https://github.com/415593725/flutter_k_chart
